ERCS/erc-7687.md

```diff
@@ -0,0 +1,245 @@
+---
+eip: 7687
+title: AUTHCALL Deposit/Withdrawal Extension for ERC-4626 Tokenized Vaults
+description: Allow Deposits and Withdrawals from Tokenized Vaults using EIP-3074 wallet signing.
+author: Señor Doggo (@fubuloubu)
+discussions-to: TBD
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-04-14
+requires: 4626, 3074
```

Comment:
Not sure how to reference an EIP as a `requires` dependency under new ERC process
```suggestion
requires: 3074, 4626
```